packages/scss/src/components/_checkbox.scss

```diff
@@ -62,6 +62,27 @@
 	}
 }
 
+// PALETTE
+// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+.checkbox {
+	@mixin checkboxColoring($palette) {
+		.checkbox-input:checked ~ .checkbox-label {
+			&::before {
+				background-color: _get($palette, "color");
+				box-shadow: inset 0 0 0 2px _get($palette, "color");
+			}
+		}
+	}
+
+	// palette colors
+	@each $name, $palette in _palettes() {
```

Comment:
see whats done for buttons [here](https://github.com/LuccaSA/lucca-front/blob/master/packages/scss/src/components/_button.scss#L60)